packages/zoslogs/package.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
I don't think we have been sourcing the package-lock.json file for individual SDKs yet.
I'm afraid I cannot recall the exact reasons and/or roadblocks we ran into at release time.
Could we keep it out for now?
Maybe we can open an issue if we need to lock dependencies at each SDK's level in the future.

Reply:
Thank you @zFernand0 , will remove it.